sites/svelte-5-preview/src/lib/Output/srcdoc/index.html

```diff
@@ -86,7 +86,8 @@
 						}
 
 						if (action === 'catch_clicks') {
-							document.body.addEventListener('click', (event) => {
+							// attached to document to not interfere with svelte's event delegators that are attached to the app root (document.body)
+							document.addEventListener('click', (event) => {
```

Comment:
Wouldn't it be better to use `on` from `svelte/events`?

Reply:
Since `srcdoc/index.html`  is used as is (imported with vite's ?raw and then set as a string as the iframes srcdoc) I had no good idea for how to import `on` here. 
At least none that did not involve setting a global variable or doing a big refactor.. given that this is a pretty one-off use I thought this might be the most reasonable fix.
But if you have any ideas / advice for how to easily use `on` here, please let me know!

Reply:
If this works without doing that then I guess it's likely fine.